soc/intel/intel_adsp/ace/Kconfig.soc

```diff
@@ -32,9 +32,9 @@ config SOC_SERIES
 config SOC_TOOLCHAIN_NAME
 	default "intel_ace15_mtpm" if SOC_INTEL_ACE15_MTPM
 	default "intel_ace15_mtpm" if SOC_INTEL_ACE20_LNL
-	default "intel_ace30_ptl" if SOC_INTEL_ACE30_PTL
+	default "intel_ace30" if SOC_INTEL_ACE30
 
 config SOC
 	default "ace15_mtpm" if SOC_INTEL_ACE15_MTPM
 	default "ace20_lnl" if SOC_INTEL_ACE20_LNL
-	default "ace30_ptl" if SOC_INTEL_ACE30_PTL
+	default "ace30" if SOC_INTEL_ACE30
```

Comment:
these should match `SOC_*` should be the value of * in the string

Reply:
In my opinion it is ok, if you see other platforms the names are entered without soc_* just simply string. For other manufacturers I randomly checked and also they don't all use soc_* names but just the name of the specific chip

Reply:
I mean this should either be `default "intel_ace30" if SOC_INTEL_ACE30` or `default "ace30" if SOC_ACE30` if not then this will fail at some point in the future when the build system generates these symbols automatically from the soc.yml files

Reply:
Oki, I understand, but I still don't think it's a problem. The SOC is selected when we have a given board declared. I think it should be added as it is, and note that other intel and non intel platforms also give the name alone, without the company name prefix

Reply:
Oki, I changed to  ```"ace30" if SOC_ACE30```